libraries/radwordsprocessing/cross-platform.md

```diff
@@ -23,6 +23,9 @@ In order to use the model of the **RadWordsProcessing** library in your cross-pl
 * **Telerik.Documents.Flow.dll**
 * **Telerik.Zip.dll**
 
+If you need to import DOC files, you will need to refer the following assembly:
```

Comment:
This section would be easier for the users to read if we add the information in a table